adapters/src/main/java/io/realm/RealmBaseAdapter.java

```diff
@@ -61,6 +64,20 @@ public void onChange(BaseRealm results) {
         }
     }
 
+    public RealmBaseAdapter(@Nullable OrderedRealmCollection<T> data) {
+        this.adapterData = data;
```

Comment:
@cmelchior I don't understand why everywhere else you haven't done that though, even on the RecyclerVIewAdapter constructor there is no isManaged() check